pants-plugins/internal_plugins/test_lockfile_fixtures/rules.py

```diff
@@ -82,12 +82,8 @@ async def collect_fixture_configs(
     pytest_pex, requirements_pex, prepared_sources, root_sources = await MultiGet(
         Get(
             Pex,
-            PexRequest(
-                output_filename="pytest.pex",
-                requirements=pytest.pex_requirements(),
-                interpreter_constraints=interpreter_constraints,
-                internal_only=True,
-            ),
+            PexRequest,
+            pytest.to_pex_request(interpreter_constraints=interpreter_constraints),
         ),
```

Comment:
should also work with just
```python
  Get(
    Pex,
    pytest.to_pex_request(interpreter_constraints=interpreter_constraints),
  ),
```
but may want to keep as-is to make the Get param type clear to the reader..? ;)

Reply:
It doesn't work, you get:

```
pants.base.exceptions.RuleTypeError: /Users/benjyw/src/pants/pants-plugins/internal_plugins/test_lockfile_fixtures/rules.py:85: Could not resolve type for `pytest.to_pex_request` in module internal_plugins.test_lockfile_fixtures.rules.

Return type annotation required for `to_pex_request` in /Users/benjyw/src/pants/src/python/pants/backend/python/subsystems/python_tool_base.py:276
```

Even though `to_pex_request` does have a return type annotation.

Reply:
#18361